eslint-example/README.md

```diff
@@ -0,0 +1,7 @@
+# ESLint Example
+
+## What it shows
+
+A project configured to use the [Mozilla ESlint configuration](https://www.npmjs.com/package/eslint-plugin-mozilla)
```

Comment:
I think another sentence will be helpful: "By using a linter on your extension source code you can be sure that you will only be writing JavaScript that is compatible with Firefox."